spec/player_spec.rb

```diff
@@ -84,5 +85,19 @@
     end
   end
 
+  describe "#draw_tiles" do
+    it "returns a collection of letters for the player" do
+      expect(@ada.draw_tiles().length).to eq 7
+    end
+  end
+
+  describe "#tiles" do
+    it "returns values in the tile_tray" do
```

Comment:
This is not a very meaningful test for the `tiles` method, because we're directly setting that value on the line above.

A better test might be to modify the `@tile_tray` instance variable indirectly by using `play` and `draw_tiles` and see how that affects the result of the `tiles` method.